llvm/lib/Target/AMDGPU/SIInstrInfo.cpp

```diff
@@ -8978,7 +8978,7 @@ unsigned SIInstrInfo::getInstSizeInBytes(const MachineInstr &MI) const {
     return getInlineAsmLength(AsmStr, *MF->getTarget().getMCAsmInfo(), &ST);
   }
   default:
-    if (MI.isMetaInstruction())
+    if (MI.isMetaInstruction() || MI.isDebugInstr())
```

Comment:
I'd expect DescSize to be correct for debug insts

Reply:
I am not sure downstream staging branch has adequate testing for this assumption.